AUTHORS

```diff
@@ -163,9 +163,6 @@ Using OpenAL Soft distributed under the GNU LGPL.
 Using MaxMind GeoIP2 .NET API distributed under
 the Apache 2.0 license.
 
-Using GeoLite2 data created by MaxMind and
-distributed under the CC BY-SA 3.0 license.
```

Comment:
Not sure about the removal here, because it is still used on the server side and the CC license requires the CC license to be mentioned and linked somewhere.

Reply:
We credit the API in the section above, and my interpretation of https://dev.maxmind.com/geoip/geoip2/geolite2/#License is that https://github.com/OpenRA/OpenRA/pull/17575/files#diff-0871ba7519d8b084807b23cec86099d8R86 is more appropriate because we are not shipping, advertising or documenting the database in the credits or elsewhere in the game client.